src/work-allocation-2/containers/work-case-list/work-case-list.component.scss

```diff
@@ -0,0 +1,119 @@
+.actions {
+  display: flex;
+  justify-content: flex-end;
+
+  .action {
+    padding: 2px;
+  }
+}
+/**
+ * Used to override the border-bottom style being set by the GDS .govuk-table__cell
+ */
+.border-none {
+  border: 0px;
+}
+
+.reset-sort-button {
+  float: right;
+  margin-bottom: -10px;
+  margin-top: -10px;
+}
+
+.column-sort {
+  padding-right: 20px;
+
+  &:hover{
+    padding-right: 20px;
+  }
+
+  &:after{
```

Comment:
This could combined into:
```
&:after, &:before {
  right: 0.5px;
}
```

By the way: `0.5px`? 🤨

Reply:
I've changed it, however clearly the CSS needs tidying up at some point because 0.5px is ridiculous.